backend/__tests__/userPostBookProperty.test.js

```
@@ -0,0 +1,371 @@
+        isValid: jest.fn()
+      }
+    }
+  };
+});
+
+// Import the controller after mocking dependencies
+// This is important to ensure mocks are set up before module import
+const { bookProperty } = require('../controllers/userControllers');
+
+describe('Book Property Controller', () => {
+  let req, res;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)